### PR TITLE
Filebeat Azure - Add missing var definitions in the manifest files

### PR DESCRIPTION
**Pull request by narph**
Should fix https://github.com/elastic/beats/issues/16270.

Due to new checks on the configuration file keys in v7.6.0,  when running:
```
filebeat setup --module azure
```
users get:
```
Exiting: Error getting config for fileset azure/activitylogs: Error interpreting the template of the input: template: text:2:22: executing "text" at <.connection_string>: map has no entry for key "connection_string"
```
due to missing to define some of the vars in the manifest file.
This PR does this.

### CODE REVIEW

File: x-pack/filebeat/module/azure/activitylogs/manifest.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ var:
     default: "insights-operational-logs"
   - name: consumer_group
     default: "$Default"
+  - name: connection_string
+  - name: storage_account
+  - name: storage_account_key
+
```
Comment (kaiyan-sheng) [inline x-pack/filebeat/module/azure/activitylogs/manifest.yml]: nit: extra line here.

**Comment by exekias** [comment]
Thank you for fixing this so quickly! I think we should have a system test to make sure this doesn’t happen again. This would probably caught it, right? https://github.com/elastic/beats/blob/master/filebeat/tests/system/test_setup.py, probably we can load it also from x-pack Filebeat.

Please update the description, even if the original issue has lots of details, it’s nice to have a good summary here explaining the change.

**Comment by andrewkroh** [comment]
@exekias I figured that the existing module tests would have caught this case. But what I realized is that because we only use the `file` input in tests that the separate config template `azure-eventhub.yml` is never exercised in tests. So we'd need a test that minimally instantiates the module with that input. Just enough to verify that it can start.

Or maybe the the `fileset` package can expose an easy to use way to evaluate a module's config templates. Then you could run assertions on the generated config output based on various inputs settings.

**Comment by exekias** [comment]
SGTM, let's get this in for the next release and work on the testing part later

**Comment by narph** [comment]
> @narph Under configs in each azure fileset like https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/azure/activitylogs/config/azure-eventhub.yml, the parameters that are not required (like connection_string, storage_account) needs to be checked to see if they are defined first. For example,
> 
> ```
> {{ if .connection_string }}
> connection_string: {{ .connection_string }}
> {{ end }}
> ```
> 
> Please see Andrew's comment in a similar PR: [#15656 (comment)](https://github.com/elastic/beats/pull/15656#discussion_r368157220)


@kaiyan-sheng , all the 3 params are required in the azure module for each fileset in this case. Two of these options might not be mandatory in the azure-eventhub input in the future so I would not add the `{{ if }}`  in order for the template to fail.

**Comment by narph** [comment]
jenkins test this

**Comment by narph** [comment]
jenkins test this

**Pull request merged**